kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/ServicesUi.java

```diff
@@ -252,6 +257,7 @@ public void onSuccess(Void result) {
                             setDirty(false);
                             ServicesUi.this.entryClass.fetchAvailableServices(null);
                             EntryClassUi.hideWaitModal();
+                            Window.Location.reload();
```

Comment:
Since we want to refresh the entire page, I think we can safely clear the body of the `onSuccess()` callback here, leaving only `Window.Location.reload()` inside it. This should spare a couple of RPC calls performed by `fetchAvailableServices()`